### ISSUE TITLE
Random folders on root dir

### ISSUE THREAD

**Issue by eoghan2t9** [sureshfizzy/CineSync#7]
This morning I logged into my server and randomly did an ls -la on my root folder / and noticed that there was 6 new folders made named Season 1-6 and looked inside them and found random symlinks for various different series in the one folder outside what I have set with in the library.sh file so not sure how they got made.

![Screenshot_20240531-194206.png](https://github.com/sureshfizzy/CineSync/assets/911750/24c6d2d5-5d59-4c80-a3f3-b410e8cb499a)

![Screenshot_20240531-193833.png](https://github.com/sureshfizzy/CineSync/assets/911750/5e3c3db1-75e8-4301-977e-8beec093730e)

**Comment by sureshfizzy** [comment]
Which os are you using ? Also can you let me know if that happens with running library.sh.

**Comment by eoghan2t9** [comment]
I'm using ubuntu 20.04 and I'll check the manual run shortly

**Comment by sureshfizzy** [comment]
Okay, I recognize something now, on my first modification, there was a problem where when the complete scan was run for the second time, it moved them to the / folder. It was later rectified and fixed, so the one you're viewing is most likely an older scan. Remove them, and it will not occur again.

**Comment by scumball** [comment]
This is how my first run of this script ended up. In my destination folder I had a _shows directory and then in that was Season 1 to Season 17 folders with the full series of each show inside just loose. What did I do wrong?

**Comment by sureshfizzy** [comment]
_destination_folder
│
└── _shows
    │
    ├── Season 1
    ├── Season 2
    ├── ...
    ├── Show A
    ├── Show B
    └── ...

Something like this ?

**Comment by sureshfizzy** [comment]
Uhh.. this is wierd. Can you create a new destination folder and try running the script again. Stop the script at the beginning itself and see the output of the files.

**Comment by scumball** [comment]
I tried it yesterday, and twice today with the same results

```
`# Source directory for TV shows
show_source_dir="/user1/docker/zurg/mnt/realdebrid/shows_1080p"

# Destination directory
destination_dir="/user1/docker/cinesync"
```

**Comment by sureshfizzy** [comment]
That shouldn't happen. I hope you haven't touched this line

https://github.com/sureshfizzy/CineSync/blob/main/Scripts/library.sh#L90

Also can we connect over telegram, so that I can resolve this faster. ID: @Black_Serpent

**Comment by scumball** [comment]
I only changed the lines shown 
I don't use telegram, would prefer this method

**Comment by sureshfizzy** [comment]
Not sure, my source folders don't have _ and haven't tested that. Did you try with any other folder which doesn't have _ in between?

**Comment by scumball** [comment]
No, as my source folders have _1080p and _4k as I split my library.

**Comment by sureshfizzy** [comment]
I'm just wondering how it worked for you earlier and causing issues now. Can you run library.sh directly and see the symlink directory which is being created over there. Before running it, do remove the files present on the destination directory.

**Comment by scumball** [comment]
It never worked for me, I am not OP

**Comment by sureshfizzy** [comment]
Sorry Mb, i didn't check the name i thought it was eoghan 🥲 was busy in some other works. 

So the problem might be with the _ which needs to be checked. If possible remove the _ from your org source directory and see if that works or I'll test from my end once I become available.

**Comment by scumball** [comment]
No rush, I don't want to change my rclone mounts so I'll see if you can test

**Comment by sureshfizzy** [comment]
Alright, I'll check the flow and let you know.

**Comment by sureshfizzy** [comment]
@scumball I'm able to reproduce this issue, the problem lies within the usage of _, I'll drop a fix asap

**Comment by sureshfizzy** [comment]
@scumball pull the code now and run the script, it will work.

**Comment by scumball** [comment]
yes, that works now. thanks

**Comment by sureshfizzy** [comment]
Closing this issue since all were resolved.
